machine/machine.go

```diff
@@ -905,15 +984,123 @@ type Translate struct {
 // It is incredibly helpful for debugging at startup and detecting
 // corrupted page tables.
 // N.B.: on x86 it appears to ignore vcpufd.
+// And, further, it always says the address is valid.
```

Comment:
What a mystery.  Well, it is used only for debugging purposes, so it is fine for now.

Reply:
yeah, it's just weird, I will keep looking.